terra-workspace-manager-jmeter/src/main/java/automation/jmeter/CreateWorkspaceSimulation.java

```diff
@@ -0,0 +1,307 @@
+        props.addItem(influxdbMetricsSender);
+        props.addItem(influxdbUrl);
+        props.addItem(application);
+        props.addItem(measurement);
+        props.addItem(summaryOnly);
+        props.addItem(samplersRegex);
+        props.addItem(percentiles);
+        props.addItem(testTitle);
+        props.addItem(eventTags);
+
+        Arguments arguments = new Arguments();
+        arguments.setName("arguments");
+        arguments.setEnabled(true);
+        arguments.setProperty(TestElement.TEST_CLASS, Arguments.class.getName());
+        arguments.setProperty(TestElement.GUI_CLASS, ArgumentsPanel.class.getName());
+        arguments.setProperty(props);
+
+        backendListener.setArguments(arguments);
+    }
+
+    private static HashTree configureTestPlan() {
+        testPlanTree = new HashTree();
+        testPlanTree.add(testPlan);
+
+        HashTree setUpThreadGroupTree = new HashTree();
+        setUpThreadGroupTree.add(threadGroup);
+        setUpThreadGroupTree.add(setupThreadGroup);
+
+        HashTree parametersTree = new HashTree();
+        parametersTree.add(headerManager);
+        parametersTree.add(userParameters);
+        parametersTree.add(backendListener);
+
+        HashTree httpSamplerTree = new HashTree();
+        httpSamplerTree.add(httpSamplerProxy);
+        httpSamplerTree.add(httpSamplerProxy, parametersTree);
+
+        setUpThreadGroupTree.add(threadGroup, httpSamplerTree);
+
+        HashTree jsr223PreProcessorTree = new HashTree();
+        jsr223PreProcessorTree.add(jsr223PreProcessor);
+        setUpThreadGroupTree.add(setupThreadGroup, jsr223PreProcessorTree);
+
+        testPlanTree.add(testPlan, setUpThreadGroupTree);
+
+        return testPlanTree;
+    }
+
+    public static void main(String[] argv) throws Exception {
+
+        // for capturing HTTP traffic
+        // System.setProperty("http.proxyHost", "127.0.0.1");
+        // System.setProperty("http.proxyPort", "8888");
+        // for capturing HTTPS traffic
+        // System.setProperty("https.proxyHost", "127.0.0.1");
+        // System.setProperty("https.proxyPort", "8888");
+
+        File jmeterHome = new File("/apache-jmeter-5.3");
+        String slash = System.getProperty("file.separator");
+
+        File jmeterProperties = new File(jmeterHome.getPath() + slash + "bin" + slash + "jmeter.properties");
+
+        // JMeter initialization (properties, log levels, locale, etc)
+        JMeterUtils.setJMeterHome(jmeterHome.getPath());
+        JMeterUtils.loadJMeterProperties(jmeterProperties.getPath());
+        JMeterUtils.initLogging();// you can comment this line out to see extra log messages of i.e. DEBUG level
+        JMeterUtils.initLocale();
+
+        initializeTestPlan("Terra Workspace Manager Test Plan");
+        setUpLoopController();
+        setUpThreadGroup();
+        setJSR223PreProcessor();
+        setLoopController(1);
+        setThreadGroup(2, 2);
+        setHttpSampler("POST", "api/workspaces/v1");
```

Comment:
Endpoint can be moved to a properties file accessible by all tests.

Reply:
I think we can also parse the swagger.yaml (just like one of the above comments related to request body) to get the API Path using the name of the API as key (CreateWorkspace).

Reply:
Added a TODO for this recommendatiuon.